frontend/src/pages/addPubVisit/AddPubVisitView.tsx

```diff
@@ -0,0 +1,65 @@
+// AddPubVisitView.tsx
+import React, { FC } from "react";
+import "./AddPubVisit.scss";
+import InputField from "../../components/input/InputFieldView";
+import PageTitleView from "../../components/pagetitle/PageTitleView";
+
+interface AddPubVisitViewProps {
+  values: PubVisit;
+  introText: string;
+  handleChange: (event: React.ChangeEvent<HTMLInputElement>) => void;
+  handleCommentChange: (event: React.ChangeEvent<HTMLTextAreaElement>) => void;
+  handleSubmitClick: () => void;
+}
+
+export interface PubVisit {
+  pubName: string;
+  visitDate: string;
```

Comment:
Nice but have you tried using a [Date type](https://www.typescriptlang.org/docs/handbook/2/basic-types.html#explicit-types) for the `visitDate`?

Reply:
I initially had a date type there, but it gave me a few annoying issues, it was easier to just save it as a string since we currently dont use the date type anyway, in the future we might want to sort the visits based on date, then we can just change it to date, but currently we no plans on adding any sorting to the page so string is sufficient for now